R/expect_null_linter.R

```diff
@@ -0,0 +1,44 @@
+#' @describeIn google-linters Require usage of expect_null(x) over expect_equal(x, NULL) and similar
+#' [testthat::expect_null()] exists specifically for testing for `NULL` objects.
```

Comment:
are we allowed to link to Suggested packages? 🤔

Reply:
I _think_ we are OK:

https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Cross_002dreferences